<ui version="4.0" >
 <class>Organizer</class>
 <widget class="QWidget" name="Organizer" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>580</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="midLineWidth" >
      <number>0</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="opaqueResize" >
      <bool>true</bool>
     </property>
     <widget class="QTabWidget" name="tabWidget" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Fixed" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="baseSize" >
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="tabPosition" >
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Albums</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5" >
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4" >
          <item>
           <widget class="QTreeView" name="trw_albums" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="contextMenuPolicy" >
             <enum>Qt::ActionsContextMenu</enum>
            </property>
            <property name="editTriggers" >
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="headerHidden" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" >
            <item>
             <widget class="QToolButton" name="tb_reload_collection" >
              <property name="text" >
               <string>...</string>
              </property>
              <property name="icon" >
               <iconset resource="../musorg.qrc" >
                <normaloff>:/icons/icons/reload.png</normaloff>:/icons/icons/reload.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tb_import_albums" >
              <property name="text" >
               <string>...</string>
              </property>
              <property name="icon" >
               <iconset resource="../musorg.qrc" >
                <normaloff>:/icons/icons/add.png</normaloff>:/icons/icons/add.png</iconset>
              </property>
              <property name="popupMode" >
               <enum>QToolButton::MenuButtonPopup</enum>
              </property>
              <property name="arrowType" >
               <enum>Qt::NoArrow</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tb_remove_albums" >
              <property name="text" >
               <string>...</string>
              </property>
              <property name="icon" >
               <iconset resource="../musorg.qrc" >
                <normaloff>:/icons/icons/remove.png</normaloff>:/icons/icons/remove.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tb_edit_album" >
              <property name="text" >
               <string>...</string>
              </property>
              <property name="icon" >
               <iconset resource="../musorg.qrc" >
                <normaloff>:/icons/icons/edit.png</normaloff>:/icons/icons/edit.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QProgressBar" name="pb_reload" >
              <property name="value" >
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2" >
       <attribute name="title" >
        <string>Favourites</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3" >
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2" />
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QTreeView" name="trw_tracks" >
      <property name="headerHidden" >
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionEdit_Album" >
   <property name="icon" >
    <iconset resource="../musorg.qrc" >
     <normaloff>:/icons/icons/edit.png</normaloff>:/icons/icons/edit.png</iconset>
   </property>
   <property name="text" >
    <string>Edit Album</string>
   </property>
  </action>
  <action name="actionRemove_Album" >
   <property name="icon" >
    <iconset resource="../musorg.qrc" >
     <normaloff>:/icons/icons/remove.png</normaloff>:/icons/icons/remove.png</iconset>
   </property>
   <property name="text" >
    <string>Remove Album</string>
   </property>
  </action>
  <action name="actionImport_Files" >
   <property name="icon" >
    <iconset resource="../musorg.qrc" >
     <normaloff>:/icons/icons/add.png</normaloff>:/icons/icons/add.png</iconset>
   </property>
   <property name="text" >
    <string>Import Files</string>
   </property>
  </action>
  <action name="actionImport_Directory" >
   <property name="icon" >
    <iconset resource="../musorg.qrc" >
     <normaloff>:/icons/icons/add.png</normaloff>:/icons/icons/add.png</iconset>
   </property>
   <property name="text" >
    <string>Import Directory</string>
   </property>
  </action>
  <action name="actionChange_Encoding" >
   <property name="text" >
    <string>Change Encoding</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../musorg.qrc" />
 </resources>
 <connections/>
</ui>
